### DISCUSSION TITLE
Introducing JTAG2UPDI (Clone)

### DISCUSSION THREAD

**Discussion by askn37** [avrdudes/avrdude#1889] [Show and tell]
Enhanced firmware that turns the ATmega328P family into a UPDI programmer.

- https://github.com/askn37/jtag2updi

Now supports the AVR-DU family. Tested with AVRDUDE 8.0 test suite.

- https://github.com/askn37/jtag2updi/pull/1

![IMG_3841](https://github.com/user-attachments/assets/d647fc43-307a-40f8-adbc-cd25811dc440)

Although it is an outdated device, it is still nice to build an Arduino compatible board with a USB-C port.

**Comment by stefanrueger** [comment]
What a cool project!
